docs/user/guides/setup/git-repo-manual.rst

```diff
@@ -205,13 +205,7 @@ Webhook activation failed. Make sure you have the necessary permissions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 If you find this error,
-make sure your user has permissions over the repository.
-In case of GitHub,
-check that you have granted access to the Read the Docs `OAuth App`_ to your organization.
```

Comment:
Don't we still need to mention this but pointing to our GitHub Application instead?

Reply:
There are no webhook creation related errors when using the github app.

Reply:
Yeah, I don't refer to webhooks but to the authorization of our GitHub App. The user still have to grant access to it, right?

I'm taking about this https://github.com/settings/connections/applications/Iv23liE6t9Wm8swttujH

Reply:
That authorization is done when you log in. But isn't needed as long as the app is installed in the repo. But again, users are never presented with this error when using a GH app.